force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -130,6 +137,12 @@ export default class NewServiceSchedule extends LightningElement {
             serviceSchedule[apiName] = value;
         });
 
+        if (this.defaultServiceQuantity) {
```

Comment:
It doesn't appear to be saving this value.

Reply:
Confirmed that it is.